src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,47 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_OF_ELEMENTS = 10;
+    private final Object[] keys;
+    private final Object[] values;
```

Comment:
I think it`s better to use K and V types instead of Object
```suggestion
    private final K[] keys;
    private final V[] values;
```